MicrosoftFluentUI.podspec

```diff
@@ -135,12 +135,6 @@ fi', :execution_position => :before_compile }
                              "ios/FluentUI/Extensions/**/*.{swift,h}"]
   end
 
-  s.subspec 'Divider_ios' do |divider_ios|
-    divider_ios.platform = :ios
-    divider_ios.dependency 'MicrosoftFluentUI/Core_ios'
-    divider_ios.source_files = ["ios/FluentUI/Divider/**/*.{swift,h}"]
-  end
```

Comment:
assuming that htis PR didn't deelete the divder folder, that means the cocoapod consuming as framework will have both separator and dvider in the same library. in that case the subspec should still stay the same?